### DISCUSSION TITLE
Best way to remove X11vnc?

### DISCUSSION THREAD

**Discussion by Romeo1984** [guysoft/FullPageOS#588] [General]
I need to remove VNC. Trying to disable it broke loading of the GUI. Any suggestions on how to remove it would be helpful. Thanks.

**Comment by guysoft** [comment]
How did you try to disable it?
Did you try?
```
systemctl stop x11vnc.service
systemctl disable x11vnc.service
```

**Reply by Romeo1984** [reply]
Yes - This is exactly what I did. After reboot, console now boots into the text login prompt and not Chromium. Looking at the system logs, I am not seeing anything obvious.

**Reply by guysoft** [reply]
Ah ok, i know why, there was a bug that will be released in the next release (0.14.0)

https://github.com/guysoft/CustomPiOS/commit/c85213f2de73bd106607382fe5bac7996e658398

You can workaround it by ruining
```   
sudo mkdir -p /var/lib/userconf-pi
sudo touch /var/lib/userconf-pi/autologin
sudo systemctl set-default graphical.target
```

**Reply by Romeo1984** [reply]
Thanks. That worked.
